editor/plugins/asset_library_editor_plugin.cpp

```diff
@@ -1379,12 +1383,17 @@ void EditorAssetLibrary::_install_external_asset(String p_zip_path, String p_tit
 	emit_signal(SNAME("install_asset"), p_zip_path, p_title);
 }
 
-void EditorAssetLibrary::disable_community_support() {
-	support->get_popup()->set_item_checked(SUPPORT_COMMUNITY, false);
+void EditorAssetLibrary::_update_asset_items_columns() {
+	int new_columns = get_size().x / (450.0 * EDSCALE);
+	new_columns = std::max(1, new_columns);
```

Comment:
We don't use `std::max()`, we use a custom `MAX()` macro instead:

```suggestion
	new_columns = MAX(1, new_columns);
```